data/abilities.js

```diff
@@ -1089,6 +1093,7 @@ exports.BattleAbilities = {
 		onResidual: function (pokemon) {
 			if (pokemon.status && this.isWeather(['raindance', 'primordialsea'])) {
 				this.debug('hydration');
+				this.add('-activate', pokemon, 'ability: Hydration');
```

Comment:
We never really standardized. I'm fine with just saying `-ability` for all ability activations.